pandas/tests/frame/methods/test_to_records.py

```diff
@@ -489,7 +489,7 @@ def keys(self):
         df = DataFrame({"A": [1, 2], "B": [0.2, 1.5], "C": ["a", "bc"]})
 
         dtype_mappings = {
-            "column_dtypes": DictLike(**{"A": np.int8, "B": np.float32}),
+            "column_dtypes": DictLike({"A": np.int8, "B": np.float32}),
```

Comment:
this isn't right, you want
```
"column_dtypes": DictLike(A=np.int8, B=np.float32),
```

you can run this test with
```
pytest pandas/tests/frame/methods/test_to_records.py
```

Reply:
Running pytest command is giving:
```
ImportError while loading conftest 'C:\Users\Victor Momodu\Documents\Programming\Data Science\Code\pandas\pandas\conftest.py'.      
pandas\__init__.py:22: in <module>
    from pandas.compat import is_numpy_dev as _is_numpy_dev  # pyright: ignore # noqa:F401
pandas\compat\__init__.py:25: in <module>
    from pandas.compat.numpy import (
pandas\compat\numpy\__init__.py:4: in <module>
    from pandas.util.version import Version
pandas\util\__init__.py:2: in <module>
    from pandas.util._decorators import (  # noqa:F401
pandas\util\_decorators.py:14: in <module>
    from pandas._libs.properties import cache_readonly
pandas\_libs\__init__.py:13: in <module>
    from pandas._libs.interval import Interval
E   ModuleNotFoundError: No module named 'pandas._libs.interval'
```
I have checked the module that was said to be missing and it exists. I searched and found [this](https://stackoverflow.com/questions/70826479/modulenotfounderror-no-module-named-pandas-libs-interval).

I have made the required change and will be pushing it to remote.

Reply:
you need to follow the instructions here to run tests https://pandas.pydata.org/docs/dev/development/contributing_environment.html